rest/jwt_auth_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/couchbase/sync_gateway/auth"
 	"github.com/couchbase/sync_gateway/base"
+	ch "github.com/couchbase/sync_gateway/channels"
```

Comment:
in auth, we use ch to mean channels package, but in rest tester we typically don't, so let's use regular package name.

channels is a very unfortunate naming choice for a concept in go and a package name.